packages/wrangler/e2e/dev.test.ts

```diff
@@ -108,11 +108,13 @@ describe.each([
 	});
 });
 
-describe.each([
+// Skipping remote python tests because they consistently flake with timeouts
+// Unskip once remote dev with python workers is more stable
+describe.skip.each([
```

Comment:
won't this skip every test in that array? Do the remote tests still need to be commented out?

Reply:
Argh yes...

Reply:
Good catch